app/javascript/components/student/editor/TestRunSummary.tsx

```diff
@@ -86,9 +87,18 @@ export function TestRunSummary({
       return
     }
 
-    setTimeout(() => dispatch({ type: 'testRun.timeout' }), timeout)
+    timer.current = window.setTimeout(
```

Comment:
```suggestion
    timer.current = setTimeout(
```

Reply:
I get this error: Type 'Timeout' is not assignable to type 'number'.

https://stackoverflow.com/questions/45802988/typescript-use-correct-version-of-settimeout-node-vs-window/50428377

What should I do?